packages/ses/README.md

```diff
@@ -178,6 +178,82 @@ const c2 = new Compartment({}, {
 });
 ```
 
+### importHook aliases
+
+If a compartment imports a module specified as `"./utility"` but actually
+implemented by an alias like `"./utility/index.js"`, the `importHook` may
+follow redirects, symbolic links, or search for candidates using its own logic
+and return a module that has a different "response specifier" than the original
+"request specifier".
+The `importHook` may return an "alias" envelope with `alias`, `compartment`,
+and `module` properties.
```

Comment:
The example code below instead returns a record with `alias` and `specifier` properties?

What does "envelope" mean here?

Reply:
Revised to avoid “envelope” term.